internal/action/generate.go

```diff
@@ -354,6 +356,9 @@ func (s *Action) generateSetPassword(ctx context.Context, name, key, password st
 		setMetadata(sec, kvps)
 		_ = sec.Set(key, password)
 		if err := s.Store.Set(ctxutil.WithCommitMessage(ctx, "Generated password for key"), name, sec); err != nil {
+			if errors.Is(err, store.ErrMeaninglessWrite) {
```

Comment:
Do you really want to return with an error if it's an ErrMeaninglessWrite? I don't think so.

Reply:
yeah, nil is probably more well-suited. anyway, the rest of the function is not needed is the ciphertext secret is unchanged